symfony/client/src/orangehrmLeavePlugin/pages/leaveType/EditLeaveType.vue

```diff
@@ -0,0 +1,173 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text tag="h6" class="orangehrm-main-title">Edit Leave Type</oxd-text>
+
+      <oxd-divider />
+
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-input-field
+            label="Name"
+            v-model="leaveType.name"
+            :rules="rules.name"
+            required
+          />
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-group
+                :classes="{wrapper: '--status-grouped-field'}"
+                label="Is Entitlement Situational"
```

Comment:
Need to use translation strings from leave module onward. add your word to `/symfony/client/messages.en_US.json`
```
{
  "source_language": "en_US",
  "target_language": "en_US",
  "date": "2021-07-19T16:21:30Z",
  "modules": {
    ....
    ....
    "leave": {
      "version": "5.0",
      "messages": {
        ....
        ....
        "is_entitlement_situational": {
          "source": "Is entitlement situational",
          "target": "Is entitlement situational",
          "description": "By clicking the “Is Entitlement Situational” option, the leave type will be excluded from reports unless there is some activity. For example: maternity leave, jury duty leave."
        }
      }
    }
  }
}
```

and use it as `:label="$t('leave.is_entitlement_situational')"`